core/lib/sentry.js

```diff
@@ -65,12 +66,28 @@ async function init(opts) {
       dsn: SENTRY_URL,
     });
 
+    /** @type {LH.Config.Settings | LH.CliFlags} */
+    let settings = opts.flags;
+    try {
+      const gatherMode = /** @type {LH.Gatherer.GatherMode} */ (
+        opts.flags.gatherMode || 'navigation');
```

Comment:
`flags.gatherMode` refers to the `-G` flag not navigation/timespan/snapshot right?